### PR TITLE
k8s java-security-usage sample enhancement 

### PR DESCRIPTION
**Pull request by liga-oz**

### CODE REVIEW

**Review by nenaraab — CHANGES_REQUESTED** [review]
unfortunately it has also merge conflicts

File: samples/java-security-usage/README.md [hunk 1]
```diff
@@ -1,7 +1,83 @@
...
+## Configure the deployment.yml
+Replace the placeholders for image repository in deployment.yml with the one created in the previous step.
+
+## Deploy the application
+Deploy the application using [kubectl cli](https://kubernetes.io/docs/reference/kubectl/)
+```shell script
+kubectl apply -f ./k8s/deployment.yml -n <YOUR NAMESPACE>
+```
+
+## Cockpit administration tasks: Assign Role Collection to your user
```
[the remark this answers is not in the document]
Comment (liga-oz) [reply samples/java-security-usage/README.md:39]: yap

File: samples/java-security-usage/README.md [hunk 2]
```diff
@@ -1,7 +1,83 @@
...
+![](../images/SAP_CP_Cockpit_AssignRoleCollectionToUser.png)
+
+Further up-to-date information you can get on sap.help.com:
+- [Maintain Role Collections](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/d5f1612d8230448bb6c02a7d9c8ac0d1.html)
+- [Maintain Roles for Applications](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/7596a0bdab4649ac8a6f6721dc72db19.html).
+
+## Access the application
+- Get an access token via `curl`. You can get the information to fill the placeholders from the service binding secret:
+```shell script
+kubectl get secret "xsuaa-service-binding" -o go-template='{{range $k,$v := .data}}{{"### "}}{{$k}}{{"\n"}}{{$v|base64decode}}{{"\n\n"}}{{end}}' -n <YOUR NAMESPACE>
```
Comment (nenaraab) [inline samples/java-security-usage/README.md:51]: what If I have already deployed another sample in the dev pod, which has a `xsuaa-service-binding` defined?

Comment (liga-oz) [reply samples/java-security-usage/README.md:51]: it would use this binding `xsuaa-service-binding` and not create a new one

**Review by nenaraab — APPROVED** [review]

**Pull request merged**